stac_fastapi/elasticsearch/stac_fastapi/elasticsearch/core.py

```diff
@@ -316,13 +315,14 @@ def create_item(self, item: stac_types.Item, **kwargs) -> stac_types.Item:
             processed_items = [
                 bulk_client.preprocess_item(item, base_url) for item in item["features"]
             ]
-            return_msg = f"Successfully added {len(processed_items)} items."
-            self.database.bulk_sync(processed_items)
+            self.database.bulk_sync(
+                processed_items, refresh=kwargs.get("refresh", False)
+            )
 
-            return return_msg
+            return None
         else:
             item = self.database.prep_create_item(item=item, base_url=base_url)
-            self.database.create_item(item=item, base_url=base_url)
+            self.database.create_item(item, refresh=kwargs.get("refresh", False))
```

Comment:
I haven't wired this through to the API yet, but calling this with "refresh=true" will cause it to refresh the index, so the item is immediately visible to a subsequent search

Reply:
Nice